src/main/java/mate/academy/internetshop/dao/impl/UserDaoImpl.java

```diff
@@ -43,4 +43,9 @@ public boolean delete(Long id) {
         return Storage.users
                 .removeIf(user -> user.getId().equals(id));
     }
+
+    @Override
+    public Optional<User> findByLogin(String login) {
+        return Storage.users.stream().filter(l -> l.getLogin().equals(login)).findFirst();

```

Comment:
```suggestion
        return Storage.users.stream()
             .filter(l -> l.getLogin().equals(login))
             .findFirst();

```
не забуваємо за формат запису стрімів